src/tests/fixtures/network-manifest.yaml

```diff
@@ -124,16 +125,19 @@ devnet_settings:
       wallet: wallet_1
       slots: 2
       btc_address: mr1iPkD9N3RJZZxXRk7xF9d36gffa6exNC
+      auto_extend: ~
```

Comment:
What does `~` mean here?

Reply:
I actually didn't know this, I copied it from the generated expected value, but a `~` is `null` for yaml: https://yaml.org/spec/1.2-old/spec.html#id2805071

Good catch, I learned something new!